src/social-network/defaults/injectCommentBox.tsx

```diff
@@ -5,9 +5,19 @@ import { useValueRef } from '../../utils/hooks/useValueRef'
 import { CommentBox } from '../../components/InjectedComponents/CommentBox'
 import Services from '../../extension/service'
 import { renderInShadowRoot } from '../../utils/jss/renderInShadowRoot'
+import { dispatchCustomEvents, selectElementContents, sleep } from '../../utils/utils'
 
-export function injectCommentBoxDefault(onPasteToCommentBox: (encryptedComment: string, current: PostInfo) => void) {
-    return function injectPostBox(current: PostInfo) {
+const defHandler = async (encryptedComment: string, current: PostInfo) => {
+    const root = current.rootNode
+    selectElementContents(root.querySelector('[contenteditable]')!)
+    dispatchCustomEvents('paste', encryptedComment)
```

Comment:
Not every website are using React. This shouldn't be the default.

Reply:
need a todo...?